render_event_consts.go

```diff
@@ -9,35 +10,57 @@ func (renderer *Renderer) Render_event_consts() ([]byte, error) {
 	f.WriteLine(`// GENERATED FILE: DO NOT EDIT!`)
 	f.WriteLine(``)
 	f.WriteLine(`package ` + renderer.Package)
-	var constants []string
+	var constants struct {
+		name  []string
+		value []string
+	}
 	for _, namedAny := range renderer.Model.ComponentsSpecificationExtension {
-		constants = append(constants, namedAny.Value.Yaml)
+		constants.name = append(constants.name, namedAny.Name)
+		constants.value = append(constants.value, namedAny.Value.Yaml)
 	}
-	f.WriteLine(`const (`)
 
 	//generate constants FactStreamType
-	if len(constants) > 0 {
-		for _, constant := range splitLineConstants(constants[0]) {
-			f.WriteLine(constant + `FactStreamType` + `=` + `"` + constant + `_facts"`)
+	if len(constants.value) > 0 {
+		constant, err := splitLineConstants(constants.value[0], constants.name[0])
+		if err != nil {
+			return f.Bytes(), err
+		}
+		f.WriteLine(`const (`)
+		for _, v := range constant {
+			f.WriteLine(v + `FactStreamType` + `=` + `"` + v + `_facts"`)
 		}
 	}
 	f.WriteLine(`)`)
-	f.WriteLine(`const (`)
 
 	//generate constants CommandsStreamType
-	if len(constants) >= 1 {
-		for _, constant := range splitLineConstants(constants[1]) {
-			f.WriteLine(constant + `CommandsStreamType` + `=` + `"` + constant + `_commands"`)
+	if len(constants.value) >= 1 {
```

Comment:
create to constants with needed property key and just take neede value from map.